### PR TITLE
libbpf-tools: update ppc vmlinux.h with bpf_perf_link, BPF_LINK_TYPE_PERF_EVENT

### PR DESCRIPTION
**Pull request by natoscott**
A BCC-based project I'm working on builds fine on x86_64 but fails on ppc. The cause is the slightly outdated vmlinux header for the latter platform, which is missing the bpf_perf_link structure and related macros.

I've regenerated a header using this following command (kernel 5.14.0) and updated the symlink:
bpftool btf dump file /sys/kernel/btf/vmlinux format c > vmlinux_5140.h

Signed-off-by: Nathan Scott <nathans@redhat.com>

### CODE REVIEW

File: libbpf-tools/powerpc/vmlinux.h [hunk 1]
```diff
@@ -1 +1 @@
-vmlinux_510.h
\ No newline at end of file
+vmlinux_5140.h
```
Comment (chenhengqi) [inline libbpf-tools/powerpc/vmlinux.h:1]: Should be `vmlinux_514.h`.

Comment (natoscott) [reply libbpf-tools/powerpc/vmlinux.h:1]: Kernel version used was 5.14.0 though, not 5.1.4 ... so I think its correct as-is, no?

Comment (natoscott) [reply libbpf-tools/powerpc/vmlinux.h:1]: Perhaps a better scheme would be something like 'vmlinux_5_14_0.h'?

Comment (chenhengqi) [reply libbpf-tools/powerpc/vmlinux.h:1]: See the README file. 505 for 5.5, let's follow this convention.

Comment (natoscott) [reply libbpf-tools/powerpc/vmlinux.h:1]: Aha - thanks for the pointer - will update.

**Pull request merged**